suite-native/module-settings/src/screens/SettingsCoinEnablingScreen.tsx

```diff
@@ -34,26 +53,42 @@ export const SettingsCoinEnablingScreen = () => {
             };
         }, [dispatch, enabledNetworkSymbols.length]),
     );
+    const transparentColor = hexToRgba(utils.colors.backgroundSurfaceElevation0, 0.001);
 
     return (
         <Screen
             screenHeader={
-                <ScreenSubHeader
-                    content={translate('moduleSettings.coinEnabling.settings.title')}
-                />
+                <View style={applyStyle(headerWrapperStyle)}>
+                    <ScreenSubHeader
+                        content={translate('moduleSettings.coinEnabling.settings.title')}
```

Comment:
Is it possible to use `Translation` component here as well?

Reply:
unfortunately no